package.json

```
@@ -1,95 +1,95 @@
-    "icon": "https://res.cloudinary.com/rakinar2/image/upload/v1659628446/SudoBot-new_cvwphw.png",
-    "readme": "https://github.com/onesoft-sudo/sudobot#readme",
-    "author": {
-        "name": "Ar Rakin",
-        "email": "rakinar2@onesoftnet.eu.org",
-        "url": "https://virtual-designer.github.io"
-    },
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/onesoft-sudo/sudobot"
-    },
-    "bugs": {
-        "url": "https://github.com/onesoft-sudo/sudobot/issues",
-        "email": "sudobot@onesoftnet.eu.org"
-    },
-    "dependencies": {
-        "@discordjs/builders": "^0.13.0",
-        "@discordjs/rest": "^0.4.1",
-        "axios": "^0.27.2",
-        "bcrypt": "^5.0.1",
-        "bufferutil": "^4.0.6",
-        "cors": "^2.8.5",
-        "date-fns": "^2.29.1",
-        "discord-api-types": "^0.32.1",
-        "discord-modals": "^1.3.6",
-        "discord.js": "^13.7.0",
-        "dot-object": "^2.1.4",
-        "dotenv": "^16.0.1",
-        "express": "^4.18.1",
-        "express-rate-limit": "^6.6.0",
-        "express-router": "^0.0.1",
-        "express-validator": "^6.14.2",
-        "jsonwebtoken": "^8.5.1",
-        "mongoose": "^6.5.4",
-        "ms": "^2.1.3",
-        "pm2": "^5.2.0",
-        "randomstring": "^1.2.2",
-        "reflect-metadata": "^0.1.13",
-        "ts-deepmerge": "^4.0.0",
-        "uuid": "^9.0.0",
-        "zod": "^3.17.2"
-    },
-    "devDependencies": {
-        "@commitlint/cli": "^17.0.3",
-        "@commitlint/config-conventional": "^17.0.3",
-        "@types/bcrypt": "^5.0.0",
-        "@types/cors": "^2.8.12",
-        "@types/dot-object": "^2.1.2",
-        "@types/express": "^4.17.13",
-        "@types/jsonwebtoken": "^8.5.9",
-        "@types/ms": "^0.7.31",
-        "@types/node": "^17.0.33",
-        "@types/randomstring": "^1.1.8",
-        "@types/uuid": "^8.3.4",
-        "@typescript-eslint/eslint-plugin": "^5.36.2",
-        "@typescript-eslint/parser": "^5.36.2",
-        "eslint": "^8.23.0",
-        "eslint-plugin-unused-imports": "^2.0.0",
-        "husky": "^8.0.1",
-        "nodemon": "^2.0.16",
-        "ts-node": "^10.9.1",
-        "typescript": "^4.6.4"
-    },
-    "scripts": {
-        "watch": "nodemon ./src/index.ts --ext ts",
-        "dev": "ts-node ./src/index.ts",
-        "flag:prod": "ts-node ./src/index.ts --prod",
-        "deploy": "node ./deploy-commands.js",
-        "start:node": "node ./build/index.js",
-        "flag:dev": "node ./build/index.js --dev",
-        "build": "tsc",
-        "clean": "bash -c 'rm -r build'",
-        "prepare": "sh -c \"husky install; sh ./init.sh\" || echo \"Error while trying to prepare project.\"",
-        "start": "pm2 start ./ecosystem.config.js --no-daemon",
-        "lint": "eslint . --ext .ts",
-        "lint:fix": "eslint . --ext .ts --fix"
-    }
-}
+  "name": "sudobot",
+  "description": "A Discord bot for moderation purposes.",
+  "version": "5.0.0",
```

Hmm looks like the auto release drafter has made this change here. This should not happen. Please commit this change here.

```suggestion
  "version": "4.0.1",
```